pom.xml

```diff
@@ -25,7 +25,7 @@
         <changelist>-SNAPSHOT</changelist>
         <hpi.compatibleSinceVersion>2.2.0</hpi.compatibleSinceVersion>
         <java.level>8</java.level>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.269</jenkins.version>
```

Comment:
it may be possible to keep compatibility but it adds quite a bit of complexity to the javascript as the dom is quite different, my first preference would be just to increase the baseline as it's safer and less complex, but up to maintainers

Reply:
```suggestion
        <jenkins.version>2.263.1</jenkins.version>
```

Reply:
that requires other changes to make the javascript compatible

Reply:
Oh, right, sorry.